.gitignore

```diff
@@ -35,3 +35,6 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+# Cloned nodejs/node for testing
+test/
```

Comment:
Nit-pick: Add a newline to be consistent with all the other files?

Reply:
Maybe `tmp` would be a better name? We may want to have unit tests one day....

Reply:
Makes sense, rebased

Reply:
Oops, missed the `tmp` comment. I'll swap that and rebase again

Reply:
OK, swapped the `test` to `tmp`

Reply:
@XhmikosR this is why it changed, but the rebase left the old empty folder refernce

Reply:
OK, I'm gonna remove it in a separate PR.

Reply:
Thanks!